userguide/config.toml

```diff
@@ -37,7 +37,7 @@ blog = "/:section/:year/:month/:day/:slug/"
 [outputs]
 home = [ "HTML" ]
 page = [ "HTML" ]
-section = [ "HTML", "RSS", "print"]
+#section = [ "HTML", "RSS", "print"]
```

Comment:
Did you intend to comment this out?

Reply:
No!   Sorry about that.   I thought I had squashed that from the commit.   (For some reason the print output was causing a hugo error.  Not sure what I did so just commented it for the time being.  Please ignore that.)